### PR TITLE
use cookie to store user login info

### PR DESCRIPTION
**Pull request by jacgrady1**
@anthonyseu @quiz1991

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -29,6 +30,10 @@ const debug = debugLib('casanova');
 
 const server = express();
 
+//must use cookieParser before expressSession
+server.use(cookieParser());
```
Comment (anthonyseu) [inline server.js]: What info will be in cookie?

Comment (jacgrady1) [inline server.js]: I am not sure. But after refreshing, the logged in user is still there. 
image

Comment (anthonyseu) [inline server.js]: 👍

Comment (anthonyseu) [inline server.js]: actually refresh won't logout user, restart server will logout all the user.

Comment (jacgrady1) [inline server.js]: @anthonyseu would that be expected?

Comment (quiz1991) [inline server.js]: For me refreshing won't logout for a while but after some time it will.

**Pull request merged**